src/scripts/todolist.js

```diff
@@ -0,0 +1,167 @@
+        <path d="M5.5 5.5A.5.5 0 0 1 6 6v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm2.5 0a.5.5 0 0 1 .5.5v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm3 .5a.5.5 0 0 0-1 0v6a.5.5 0 0 0 1 0V6z"/>
+        <path fill-rule="evenodd" d="M14.5 3a1 1 0 0 1-1 1H13v9a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V4h-.5a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1H6a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1h3.5a1 1 0 0 1 1 1v1zM4.118 4 4 4.059V13a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1V4.059L11.882 4H4.118zM2.5 3V2h11v1h-11z"/>
+      </svg>`;
+
+    taskElement.append(checkbox, taskDescription, dots, bin);
+    taskHolder.append(taskElement);
+  }
+
+  // show task
+
+  showTask() {
+    this.taskInfo.forEach((tasks) => {
+      TodolistClass.displayTask(tasks);
+    });
+  }
+
+  // updateTaskStatus
+  updateTaskStatus(chck, index) {
+    chck.addEventListener('click', (r) => {
+      if (r.currentTarget.checked) {
+        chck.nextElementSibling.classList.add('text-decoration-line-through');
+        this.taskInfo[index].completed = true;
+        localStorage.setItem('Todo', JSON.stringify(this.taskInfo));
+      } else {
+        if (
+          chck.nextElementSibling.classList.contains(
+            'text-decoration-line-through',
+          )
+        ) {
+          chck.nextElementSibling.classList.remove(
+            'text-decoration-line-through',
+          );
+        }
+        this.taskInfo[index].completed = false;
+        localStorage.setItem('Todo', JSON.stringify(this.taskInfo));
+        window.location.reload();
+      }
+    });
+  }
+
+  removeSelectedTask(trash, index) {
+    trash.addEventListener('click', () => {
+      taskHolder.removeChild(trash.parentElement);
+      this.taskInfo.splice(index, 1);
+      this.taskInfo.forEach((list) => {
+        if (list.index > index) {
+          list.index -= 1;
+        }
+      });
+      localStorage.setItem('Todo', JSON.stringify(this.taskInfo));
+      window.location.reload();
+    });
+  };
+
+   // filter
+
+  remove(cl){
+    cl.addEventListener('click', (m) => {
+      this.taskInfo = this.taskInfo.filter((cls) => cls.completed !== true );
+      this.taskInfo.forEach((remain,index) => {
+        if(remain.index !== index){
+          remain.index = index;
+        };
+      });
+      localStorage.setItem('Todo', JSON.stringify(this.taskInfo));
+      window.location.reload();
+    });
+  };
+}
+
+const tododisplay = new TodolistClass();
+
+// function to expoert
+
+function runfunction() {
+  tododisplay.showTask();
+  input.addEventListener('keypress', (e) => {
+    if (input.value && e.key === 'Enter') {
+      const newaddedTask = tododisplay.addTask(input.value);
+      TodolistClass.displayTask(newaddedTask);
+      input.value = '';
+      e.preventDefault();
+      window.location.reload();
+    }
+  });
+
+  const dots = document.querySelectorAll('.dots');
+
```

Comment:
### Hello @ijosue16 
Great job working on this project.
- I have another suggestion for this lines of code, it would be great if you can remove the extra spaces from your code as this will make it more readable and easily understandable.